src/web/app/pages-session/session-submission-page/session-submission-modals.component.ts

```diff
@@ -0,0 +1,158 @@
+import { Component, Input } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+
+/**
+ * Modal to inform the completion of the saving process
+ */
+@Component({
+  template: `
+    <div class="modal-header text-white"
```

Comment:
Let's put this out into an HTML file. It's always good to have a separate HTML file for each controller, especially if the template is long like this.